### PR TITLE
Add relations

### PR DESCRIPTION
**Pull request by Silent777**
add basic view
add basic urls
new migration
modify .pylintrc

## Code reviewers

- [x] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [x] @lhalam

## CHECK LIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/task/models.py [hunk 1]
```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
...
     updated_at = models.DateTimeField(auto_now=True)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
+    users = models.ManyToManyField(CustomUser)
+
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        task instance object.
+
+        :return: task title, task description, task status
```
Comment (redmyt) [inline eventually/task/models.py]: update docstring's `return` statement according to function's return.

File: eventually/task/models.py [hunk 2]
```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
...
+        """
 
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,
+                                                [user.id for user in self.users.all()])
```
Comment (redmyt) [inline eventually/task/models.py]: maybe this field could return enough long output.

File: eventually/task/models.py [hunk 3]
```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
...
     def __str__(self):
         """
         Magic method that returns string representation of
-        event instance object.
+        task instance object.
 
         :return: task title, task description, task status
```
Comment (redmyt) [inline eventually/task/models.py]: update docstring's `return` statement according to function's return.

File: eventually/task/models.py [hunk 4]
```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
...
 
-        return "{} {} {}".format(self.title, self.description, self.status)
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,
+                                                [user.id for user in self.users.all()])
```
Comment (redmyt) [inline eventually/task/models.py]: maybe this field could return enough long output.

File: eventually/task/models.py [hunk 5]
```diff
@@ -67,16 +100,20 @@ def to_dict(self):
             'status': 1,
             'created_at': 1509540116,
             'updated_at': 1509540116,
+            'event' : 13,
+            'users' : [21, 33]
```
Comment (redmyt) [inline eventually/task/models.py]: maybe it is a good idea to write each dict's field from a new line using `|` char.

File: eventually/task/models.py [hunk 6]
```diff
@@ -67,16 +100,20 @@ def to_dict(self):
...
-            'id':self.id,
+            'id': self.id,
             'title': self.title,
             'description': self.description,
             'status': self.status,
             'created_at': int(self.created_at.timestamp()),
             'updated_at': int(self.updated_at.timestamp()),
+            'event': self.event.id,
+            'users': [user.id for user in self.users.all()]
         }
```
Comment (makarman) [inline eventually/task/models.py]: it is better to write fields just under "{" . And put "}" behind "]" like this "'users': [user.id for user in self.users.all()]}"

**Review by makarman — CHANGES_REQUESTED** [review]

File: eventually/task/models.py (hunk 2 above)
Comment (Silent777) [reply eventually/task/models.py]: you think more then 3 - 6 id?

File: eventually/task/models.py (hunk 5 above)
Comment (Silent777) [reply eventually/task/models.py]: ok

File: eventually/task/models.py (hunk 2 above)
Comment (redmyt) [reply eventually/task/models.py]: is this info necessary? But if it is, there is no problem.

File: eventually/task/models.py [hunk 7]
```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
...
+        :return: task title, task description, task status
+        """
 
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,
```
Comment (makarman) [inline eventually/task/models.py]: you have event = models.ForeignKey(CustomUser, null=True)
so add self.event.id if self.owner else None,

File: eventually/task/models.py (hunk 2 above)
Comment (makarman) [inline eventually/task/models.py]: [users.id for musers in self.users.all()] if self.users else None)

File: eventually/task/models.py (hunk 6 above)
Comment (Silent777) [reply eventually/task/models.py]: JS )))

File: eventually/task/models.py (hunk 7 above)
Comment (Silent777) [reply eventually/task/models.py]: I have  event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
owner??? 
there is no such field in my model :)

File: eventually/task/urls.py [hunk 8]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.TaskView.as_view()),
+    url(r'^(?P<task_id>\d+)/$', views.TaskView.as_view()),
+]
```
Comment (lhalam) [inline eventually/task/urls.py]: add endline

File: eventually/task/models.py [hunk 9]
```diff
@@ -96,14 +133,19 @@ def get_by_id(task_id):
...
-    def create(title=None, description=None, status=0):
+    def create(event, users, title=None, description=None, status=0):
         """
         Static method that creates instance of Task class and creates database
         row with the accepted info.
 
+        :param: event: Certain Event's object. Is required.
+        :type event: Event model.
+
+        :param: users: Certain CustomUser's object. Is required.
```
Comment (lhalam) [inline eventually/task/models.py]: Certain tuple CustomUser's objects

File: eventually/task/urls.py (hunk 8 above)
Comment (Silent777) [reply eventually/task/urls.py]: ++ endline

File: eventually/task/models.py (hunk 9 above)
Comment (Silent777) [reply eventually/task/models.py]: ++'tuple'

**Review by makarman — DISMISSED** [review]

File: eventually/task/models.py (hunk 1 above)
Comment (Silent777) [reply eventually/task/models.py]: ok

File: eventually/task/models.py (hunk 3 above)
Comment (Silent777) [reply eventually/task/models.py]: ok

**Review by redmyt — DISMISSED** [review]

File: eventually/task/models.py (hunk 4 above)
Comment (Silent777) [reply eventually/task/models.py]: leave as it is yet

**Review by redmyt — APPROVED** [review]

**Pull request merged**